NEWS.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to this project will be documented in this file.
 
+#### [0.99.16] - 2023-06-23
+- increase comporession level (Tavor_2020.qs; < 5MB limit)
```

Comment:
typo, compression

Reply:
thanks. fixed in 9b0ba7ef87565a13e166ec1e397590c5c28daf21.